s3_to_redshift.go

```diff
@@ -26,6 +27,7 @@ var (
 	dataDate        = flag.String("date", "", "data date we should process, must be full RFC3339")
 	configFile      = flag.String("config", "", "schema & table config to use in YAML format")
 	gzip            = flag.Bool("gzip", true, "whether target files are gzipped, defaults to true")
+	timeGranularity = flag.String("granularity", "day", "how often we expect to append new data")
```

Comment:
How strongly do you feel about this? I have a preference to keep the defaults behavior, given that no use case actually exists right now for non-day granularity (and I generally prefer the principle of engineering things as the use case arises). However, if you feel strongly that we should enforce day granularity, we can absolutely add it in.